src/main/java/com/bulkbuy/enterprise/dto/Order.java

```diff
@@ -17,7 +17,7 @@
     private String orderStatus;
 
     private int itemID;
-    private String customer;
+    private String customerName;
```

Comment:
The name of the variable should describe the intent of the variable clearly
https://www.baeldung.com/java-clean-code